dist/tools/drone-scripts/build_and_test.sh

```diff
@@ -16,9 +16,16 @@ function exec_build_func {
     fi
 
     echo "Begin group ${group}"
+    # spawn new directory for the group
+    mkdir -p "build_tmp/${group}"
+    rsync -a --exclude="build_tmp" . "build_tmp/${group}"
```

Comment:
OK, sure. Could you add some exclusion for the `bin` directories as well? in case someone runs this from their workspace and already have a bunch of pre-built stuff in there.